src/components/Item/modifiedCellRenderer.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ItemModified from './ItemModified';
+import { VIEW_SEARCH } from '../../constants';
+import type { View, BoxItem } from '../../flowTypes';
+import './ModifiedCell.scss';
+
+export default (
+    rootId: string,
+    view: View,
+) => ({ rowData }: { rowData: BoxItem }) => (
+    <div className='be-item-modified'>
```

Comment:
Is this wrapping div needed? Can this div be just moved inside ItemModified and used in-place of the wrapping span that has? Or just the span used without div?